Library/Formula/megatools.rb

```diff
@@ -1,8 +1,8 @@
 class Megatools < Formula
   desc "Command-line client for Mega.co.nz"
   homepage "https://megatools.megous.com/"
-  url "https://megatools.megous.com/builds/megatools-1.9.96.tar.gz"
-  sha256 "c657a0988223aada021a7a6ed408227fc66435de7248f23952ad7adf9e1242f0"
+  url "https://megatools.megous.com/builds/megatools-1.9.97.tar.gz"
+  sha256 "3316c56ee55adef0ab113aad41ef95eb28fc15218639f69b1e04362b1c725af3"
```

Comment:
Please remove this chunk. New versions are added upstream by `Homebrew/homebrew`.